applications/MeshMovingApplication/custom_utilities/mesh_velocity_calculation.cpp

```diff
@@ -86,7 +86,7 @@ void CalculateMeshVelocities(ModelPart& rModelPart,
     for (int i=0; i<num_local_nodes; i++) {
         const auto it_node  = nodes_begin + i;
         auto& r_mesh_v0 = it_node->FastGetSolutionStepValue(MESH_VELOCITY);
-        noalias(r_mesh_v0)  = coeffs[0] * it_node->FastGetSolutionStepValue(MESH_DISPLACEMENT);
+        noalias(r_mesh_v0) += coeffs[0] * it_node->FastGetSolutionStepValue(MESH_DISPLACEMENT);
         noalias(r_mesh_v0) += coeffs[1] * it_node->FastGetSolutionStepValue(MESH_DISPLACEMENT, 1);
         noalias(r_mesh_v0) += coeffs[2] * it_node->FastGetSolutionStepValue(MESH_DISPLACEMENT, 2);
     }
```

Comment:
```suggestion
noalias(r_mesh_v0) += it_node->GetValue(AUX_MESH_VELOCITY);
        }
```

note that here we would need to check before (or access the Node as `const`) if the `AUX_MESH_VELOCITY` exists